pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -105,15 +105,15 @@ def on_train_batch_start(self, trainer, *args, **kwargs):
             interval = 'step' if self.logging_interval is None else 'any'
             latest_stat = self._extract_stats(trainer, interval)
 
-            if trainer.logger is not None and latest_stat:
+            if latest_stat:
```

Comment:
Why did you remove trainer.logger check ? What happens if Trainer(logger=False)

Reply:
we have a hard stop if there is no logger, so it's not required here.
https://github.com/PyTorchLightning/pytorch-lightning/blob/15a400b95faf45124f28fd8ca4ef7520d47df88a/pytorch_lightning/callbacks/lr_monitor.py#L75-L84